packages/x-charts/src/SparkLineChart/SparkLineChart.tsx

```diff
@@ -296,6 +314,10 @@ SparkLineChart.propTypes = {
     x: PropTypes.number,
     y: PropTypes.number,
   }),
+  /**
+   * The width of the chart in px. If not defined, it takes the width of the parent element.
+   * @default undefined
+   */
```

Comment:
cold we move this to the `height` prop?

Reply:
Not sure to get your point. Do you mean having the `width` and `height` next to each over?

If yes, the answer is no 😁 Those Proptypes are automatically generated. I can't modify their order